src/allotropy/parsers/moldev_softmax_pro/softmax_pro_data_creator.py

```diff
@@ -204,40 +204,21 @@ def _get_reduced_calc_docs(data: StructureData) -> list[CalculatedDataItem]:
     ]
 
 
-def _get_group_agg_calc_docs(
```

Comment:
Note to reviewer: this is the special case for aggregated rows, removed in favor of generalizing the other case.